client/src/components/flavor-thumbnail.module.css

```diff
@@ -0,0 +1,18 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+.container {
+  width: 200px;
+  height: 300px;
+  border: 1px solid lightgrey;
+  text-align: center;
+}
+
+.image {
+  width: 86%;
+  height: 66.6667%;
+  object-fit: contain;
+}
+
+p {
```

Comment:
heads up, the style in this p tag will apply globally, we only get the benefit of css modules if the selector is a class. makes sense for font family to be global anyway but either now or later it'd be good to move it to something like a body tag in a css file that we wanna use for global styles.

Reply:
Cool, I didn't know that! Changing.